### PR TITLE
Adding screenshot to README.md.

### PR DESCRIPTION
**Pull request by fosdickio**
Adding a simple screenshot to `README.md` so users can quickly grok what TironCalc is.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,16 +31,16 @@ Load an existing Excel file:
...
+-   `+` to add a sheet
+-   `s` to go to the next sheet
+-   `PgUp/PgDown` to navigate rows faster
 
 There is no way to save to file for now.
 
 ## Inspiration
 
-James Gosling of Java fame created [sc](https://en.wikipedia.org/wiki/Sc_(spreadsheet_calculator)) the spreadsheet calculator.
+James Gosling of Java fame created [sc](<https://en.wikipedia.org/wiki/Sc_(spreadsheet_calculator)>) the spreadsheet calculator.
```
Comment (nhatcher) [inline README.md]: I don't think we need the autolinks

**Comment by nhatcher** [comment]
I think we can build a better screenshot, right?

**Review by nhatcher — CHANGES_REQUESTED** [review]
Thanks!, this is a great idea

**Comment by fosdickio** [comment]
> I think we can build a better screenshot, right?

Ha!  Yes, I think so.  I'm using `long.xlsx` and it's a bit busy for someone looking at the repo for the first time. 😆 

I'll update this to include one that's a bit more approachable.

**Review by nhatcher — APPROVED** [review]

**Pull request merged**